packages/mwp-api-proxy-plugin/src/util/send.js

```diff
@@ -8,7 +8,7 @@ import externalRequest from 'request';
 import config from 'mwp-config';
 
 export const API_META_HEADER = 'X-Meta-Request-Headers';
-const FULL_URL_PATTERN = /^https?:\/\//;
+const IS_DIY_EDGE_URL_PATTERN = /^https?:\/\//;
```

Comment:
renaming to make the purpose more clear - we may want to update this at some point to specifically exclude https://api.meetup.com URLs, but there's no particular reason for people to specify the full URL in that case.

Reply:
Could you elaborate a bit on _why_ the DIY edge url only tests for https? Is it because the Shared Edge and other external requests don't use https? Or is it application implementation specific - `https://` full URLS are specified only for DIY Edge.

Reply:
I guess naming it DIY edge is actually more confusing because this pattern could be used for _any_ endpoint, even a third party one, so I'll update the name. Essentially, if you specify the full host (protocol + domain) in the `endpoint`, it means you do _not_ want to use the default `baseUrl` of `https://api.meetup.com`. This regex checks for the `https://` protocol at the start of the string to check whether the endpoint is pointing to a non-default host (essentially, if you specify the protocol, you're specifying a fully-qualified URL).